Prog/Sec8MF/Sec8MF_2017_08.sas

```diff
@@ -0,0 +1,54 @@
+/**************************************************************************
+ Program:  Sec8MF_yyyy_mm.sas
+ Library:  HUD
+ Project:  NeighborhoodInfo DC
+ Author:   
+ Created:  
+ Version:  SAS 9.2
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Compile Section 8 multifamily contract/project data.
+ Creates files for DC, MD, VA, and WV.
+ 
+ NB:  Change upload=Y for final batch run.
+
+ Modifications:
+  10/19/14 PAT Updated for SAS1.
+  12/23/14 PAT New version incorporating %sec8mf_readbasetbls() and
+               %Sec8MF_dmvw() macros.
+  11/29/15 PAT Upload=Y is now the default. 
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( HUD )
+%DCData_lib( RealProp )
+
+
+*--- EDIT PARAMETERS BELOW -----------------------------------------;
+
+  ** Enter date of HUD database as SAS date value, ex: '25nov2014'd **;
+
+  %let s8filedate = '01sep2017'd;
```

Comment:
I got an error when I first submitted this program because it couldn't find the input files. 

`ERROR: Physical file does not exist, L:\Libraries\HUD\Raw\Sec8MF\2017-09\MF_Properties_with_Assistance___Sec8_Contracts.txt.
`

Part of the problem was that the subfolder name for the input (.txt) files needs to align with the update extract date, that is, the input files need to be saved in **L:\Libraries\HUD\Raw\Sec8MF\2017-09**. I made that fix. 

In addition, there was still a problem because the .txt files had names that were different than what the program was expecting. The "&" in the .txt file names has to be changed to an underscore ("_"). Again, I made the fix so that I could test the program. (Perhaps in the older version of Access this happened automatically during the export, because there's nothing in the instructions about it. We may need to do it manually from now on.)